### ISSUE TITLE
desi_tile_redshifts timeouts

### ISSUE THREAD

**Issue by akremin** [desihub/desispec#1203]
The new desi_tile_redshifts was designed to work per-tile because the single large nightly jobs were timing out after 40 minutes. It has worked well to reduce each job to manageable levels, but on the first real-world trial run 3 of the 13 jobs timed out:

```
/global/cfs/cdirs/desi/spectro/redux/daily/run/scripts/tiles/80663/20210322/coadd-redshifts-80663-20210322-40965662.log
/global/cfs/cdirs/desi/spectro/redux/daily/run/scripts/tiles/80666/20210322/coadd-redshifts-80666-20210322-40965664.log
/global/cfs/cdirs/desi/spectro/redux/daily/run/scripts/tiles/80732/20210322/coadd-redshifts-80732-20210322-40965666.log
```

Those were resubmitted and took another 2-5 minutes to run. Based on this I think we need to update the requested run time from 15 minutes to 20.

**Comment by akremin** [comment]
Fixed in direct-to-master update (commit 42f6a83)

**Comment by sbailey** [comment]
Analyzing timestamps from 80663/20210322:
   * sp6 was disabled, so not included in processing
   * desi_group_spectra took <1 min for 8 spectrographs, and 7 minutes for sp7
   * desi_coadd_spectra took <1 min for 8 spectrographs, and 3 minutes for sp4
   * rrdesi took ~2 min for 8 spectrographs, and sp2 hit the job timelimit after 4 min

This smells like I/O hangs.  Each group of commands waits for all to be finished before proceeding with the next, which exacerbated the effect because other spectrographs didn't continue with the next step when they were finished, thus we were limited by the longest of individual steps.
